tests/PHPUnit/System/OneVisitorTwoVisitsWithCookieSupportTest.php

```diff
@@ -46,6 +46,18 @@ public static function getOutputPrefix()
     {
         return 'OneVisitorTwoVisits_withCookieSupport';
     }
+
+    public function provideContainerConfig()
+    {
+        return array(
+            'Piwik\Config' => \DI\decorate(function ($previous) {
+                $general = $previous->General;
+                $general['action_title_category_delimiter'] = "/";
+                $previous->General = $general;
+                return $previous;
+            }),
+        );
+    }
```

Comment:
Should we just change the tests w/ the new config or is there value in keeping the old behavior in all tests?

Reply:
Actually, it looks like only a couple of the tests have this, so I guess most use the new default?

Reply:
Most use default empty, but wanted to make sure some tests still use this behaviour to test it as well.

Reply:
👍